bundle/manifests/oadp-operator.clusterserviceversion.yaml

```
@@ -1084,6 +1084,9 @@ spec:
         serviceAccountName: velero
       deployments:
       - label:
+          app.kubernetes.io/component: controller-manager
+          app.kubernetes.io/name: oadp-operator
+          app.kubernetes.io/part-of: oadp
```

Let's standardize to the same label.
```suggestion
          app.kubernetes.io/part-of: oadp-operator
```

https://github.com/openshift/oadp-operator/blob/790106ecaa0f71eab38d011762c4dbc48fe90c48/config/prometheus/monitor.yaml#L11

---

https://github.com/search?q=repo%3Aopenshift%2Foadp-operator%20%20app.kubernetes.io%2Fpart-of&type=code